src/tests/address.service.distance.test.ts

```diff
@@ -0,0 +1,184 @@
+import addressService from "../services/address.service";
```

Comment:
I'd probably combine these test classes and differentiate between the method calls via the describe syntax.